config/management/README.md

```diff
@@ -67,3 +67,74 @@ While this is compiled as a single binary it provides several different faciliti
   storage -- leveraging the identity tool.
 * Converting a genesis configuration and a secure storage into a genesis.blob /
   genesis waypoint.
+
+## The Process
+
+* The association first sets up a secure storage, in this case, say GitHub. All
+  participants must have commit and read access to this repository. The
+  association will share out the GitHub repository and a distinct name for each
+  participant.
+* Each participant must retrieve an appropriate
+  [token](https://github.com/settings/tokens) for their account that allows
+  access to the `repo` scope. This token should be stored locally on their disk
+  in a file accessible to the management application.
+* Each participant should initialize their respective key: `association`,
+  `owner`, or `operator` in a secure storage solution, likely Vault. And
+  prepare another token file accessible to the manaagement application.
+* Each participant will upload their key to GitHub specifying their namespace
+  as their distinct name provided earlier:
+  * Association:
+```
+management association-key \\
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+    --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
+```
+  * Owner:
+```
+management owner-key \\
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+    --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
+```
+  * Operator:
+```
+management association-key \\
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+    --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
+```
+* The operator will upload a signed validator-config (note: the owner address
+  is irrelevant in this run):
+```
+management validator-config \\
+    --owner-address 00000000000000000000000000000000 \\
+    --validator-address '\\dns\\DNS\\tcp\\PORT' \\
+    --fullnode-address '\\dns\\DNS\\tcp\\PORT' \\
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+    --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
```

Comment:
What is the `owner` param? And `namespace`?

Reply:
owner would be the owner or project where the repo is held
namespace is the directory within the namespace

I'm actually thinking we should give a templatable script for this process specific to our genesis and shared out of band

Reply:
So it would actually be "libra"?